interfaces/console/src/app/console/subscribers/page.tsx

```diff
@@ -560,29 +559,10 @@ const Page = () => {
   const handleCloseTopUp = () => {
     setIsToPupData(false);
   };
-
   const handleTopUp = async (plans: { planId: string; simId: string }[]) => {
     try {
-      if (plans.length === 1) {
-        const data = {
-          sim_id: plans[0].simId,
-          package_id: plans[0].planId,
-        };
-        await addPackageToSim({
-          variables: {
-            data: {
-              sim_id: plans[0].simId,
-              package_id: plans[0].planId,
-              start_date: new Date(Date.now() + 1 * 60000).toISOString(),
-            },
-          },
-        });
-
-        await activatePackageSim({
-          variables: { data },
-        });
-      } else {
-        for (const plan of plans) {
+      for (const plan of plans) {
+        try {
           await addPackageToSim({
             variables: {
               data: {
```

Comment:
Rather calling multiple `addPackageToSim` api's here  we can move this behaviour to BFF level. Add a new mutation on BFF which takes list of packages as payload.